test/gradient/elixir_type_test.exs

```diff
@@ -0,0 +1,48 @@
+defmodule Gradient.ElixirTypeTest do
+  use ExUnit.Case
+  doctest Gradient.ElixirType
+
+  alias Gradient.ElixirType
+
+  test "pp integer()" do
```

Comment:
The "pp ..." tests seem to have the same form and only differ with regard to `expected` and `actual` values. We could use a block like the first example here - https://elixirforum.com/t/exunit-generate-test-cases-dynamically-inside-a-describe-block/29628/3?u=erszcz - to generate them.

Reply:
Awesome, thanks 😃